core/vm/interpreter.go

```diff
@@ -47,7 +47,7 @@ type Config struct {
 type Interpreter interface {
 	// Run loops and evaluates the contract's code with the given input data and returns
 	// the return byte-slice and an error if one occurred.
-	Run(contract *Contract, input []byte, static bool) ([]byte, error)
+	Run(contract *Contract, input []byte, static bool, indestructible bool) ([]byte, error)
```

Comment:
Some go syntactic sugar might be nice:

```suggestion
	Run(contract *Contract, input []byte, static, indestructible bool) ([]byte, error)
```